docs/release/trg-0/trg-1-08.md

```diff
@@ -0,0 +1,31 @@
+---
+title: TRG 1.08 - Interfaces Documentation
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History          |
+|------------|--------------|-----------------------|
+| Draft      | 31-Jan-2024  | created initial Draft |
+
+## Why
+
+Interfaces Documentation is crucial for integration testing and ensuring seamless interoperability for Developers consuming the product API
+
+## Description
+
+- The interface documentation **has to** be stored in the folder: ```docs/interfaces/```
```

Comment:
"must" or "should". 
I would prefer `docs/api`, if we put it in the `/docs` folder